client/tests/unit/components/Policy.spec.js

```diff
@@ -0,0 +1,64 @@
+import { mount } from '@vue/test-utils';
+import PolicyListItem from "../../../src/components/Policy";
+
+const factory = (values = { propsData: {} }) => mount(PolicyListItem, { ...values });
+
+describe('Policy tests', () => {
+  describe('rendering tests', () => {
+    it('should display appropriate title', () => {
+      const wrapper = factory({ propsData: { title: 'Policy Title' } });
+      expect(wrapper.find('.title').text()).toEqual('Policy Title');
+    });
+    it('should display appropriate summary', () => {
+      const wrapper = factory({ propsData: { summary: 'Policy Summary' } });
+      expect(wrapper.find('.summary').text()).toEqual('Policy Summary');
+    });
+    it('should display appropriate Intro Date', () => {
+      const wrapper = factory({ propsData: { date_introduced: '2020-06-01T05:00:00.000Z' } });
+      expect(wrapper.find('.introdate').text()).toEqual('Introduced: 6-1-2020');
+    });
+  });
+
+  describe('computed tests', () => {
+    describe('rowType tests', () => {
+      it('should return "sentiment-bar-1 odd" if row_num is 0', () => {
+        const wrapper = factory({ propsData: { row_num: 0 } });
+        expect(wrapper.vm.rowType).toEqual('sentiment-bar-1 odd');
+      });
+      it('should return "sentiment-bar-1 even" if row_num is 1', () => {
+        const wrapper = factory({ propsData: { row_num: 1 } });
+        expect(wrapper.vm.rowType).toEqual('sentiment-bar-1 even');
+      });
+      it('should return "sentiment-bar-1 odd" if row_num is 2', () => {
+        const wrapper = factory({ propsData: { row_num: 2 } });
+        expect(wrapper.vm.rowType).toEqual('sentiment-bar-1 odd');
+      });
+    });
+    describe('getIntroDt tests', () => {
+      it('should return formatted date', () => {
+        const wrapper = factory({ propsData: { date_introduced: '2020-06-01T05:00:00.000Z' } });
+        expect(wrapper.vm.getIntroDt).toEqual('6-1-2020');
```

Comment:
This is the test that helped identify that Policy.vue was returning the wrong month.

Reply:
@eis95 Do you have any more details or an Issue for this suspected bug?  I don't see a real description of it (and I don't really use the client ever).  Entering an Issue would be best to capture the bug.

For the test the input is UTC time and the output is local time so the result varies -- 5/31 or 6/1 depending on timezone. You've already seen my proposed test fix.  If the client is actually getting mixed up somewhere (input timezone issues???), then I'd like to see an issue describing it.  You can't expect the local date to always match the UTC date though.

Reply:
I agree that you can't expect the two to match. So maybe there is no bug any longer. I thought I saw that the date was displayed as 5/32 or 6/01. So I apologize for the confusion. If you'd like I can take a look at the test and see about forcing the timezone to be UTC and maybe that would solve the problem?

Reply:
5/32 was a typo for 5/31. So the result was ok for local time but the test failed. You could try to force a local time zone or detect time zone so you know which answer is correct or accept either answer (weaker but good enough?).  

What I chose was to use 11ish am UTC which is the same day around the globe except some islands that stretch the international date line to +13 +14 time zones. Not perfect, but pretty good. If you want to make the test smarter though, that'd be cool. Someone should try testing in Samoa or Fiji.